sources/packages/web/src/views/aest/student/applicationDetails/AssessmentsSummary.vue

```diff
@@ -32,22 +38,25 @@
       @viewOfferingRequest="goToOfferingRequest"
       @viewApplicationException="goToApplicationException"
       @viewScholasticStandingChange="goToScholasticStanding"
+      :key="historyKey"
```

Comment:
Same here. couldn't get it's purpose.

Reply:
https://github.com/bcgov/SIMS/pull/3042#pullrequestreview-1964565367